.pre-commit/run_forge_tests.sh

```diff
@@ -11,5 +11,7 @@ source .pre-commit/foundry_utils.sh
 
 for dir in $(foundryroots $@); do
   echo "Running 'forge test' in ./$dir"
-  (cd $dir && pnpm install && forge test && forge snapshot)
+  # fuzz tests snapshots are different in CI, so we ignore them
+  # TODO: figure out why, and fix
+  (cd $dir && pnpm install && forge test && forge snapshot --no-match-test testFuzz_syncWithOmni)
```

Comment:
so we're running the test, just not outputting the snapshot data for that test?

Reply:
yes - for now, it's just been a little annoying